caikit/core/data_model/training_status.py

```diff
@@ -40,3 +42,12 @@ def is_terminal(self):
             self.__class__.CANCELED,
             self.__class__.ERRORED,
         ]
+
+
+@dataobject(PACKAGE_COMMON)
+class TrainingInfo(DataObjectBase):
+    errors: Annotated[List[str], FieldNumber(1)]
```

Comment:
Why is this a list? Under what circumstances do we expect there to be multiple elements here?

Reply:
I think couple of examples I was thinking about was, in case we add a validation step where we collect all the data validation errors and return it to user.

Reply:
Cool, that makes sense!